### PR TITLE
K8s: Proof of Concept

### PR DESCRIPTION
**Pull request by toddtreece**
Update: this has been this be split into two PoC branches:
* #66218 
* #66219 

This is the main feature branch that all other K8s PoC related PRs merge into.

This PR will not be merged as is -- but it will aim to stay in sync with main and learn how to best integrate

`conf/custom.ini`:
```ini
app_mode = development

[feature_toggles]
k8s = true
publicDashboards = true
```

### CODE REVIEW

**Review by ryantxu — CHANGES_REQUESTED** [review]
adding a request changes note to block merging

File: Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ include .bingo/Variables.mk
 
 .PHONY: all deps-go deps-js deps build-go build-backend build-server build-cli build-js build build-docker-full build-docker-full-ubuntu lint-go golangci-lint test-go test-js gen-ts test run run-frontend clean devenv devenv-down protobuf drone help gen-go gen-cue fix-cue
 
-GO = go
+GO = CGO_CFLAGS="-DSQLITE_ENABLE_DBSTAT_VTAB=1 -DSQLITE_USE_ALLOCA=1" go
```
Comment (ryantxu) [inline Makefile:12]: required for kine... but not ideal

**Comment by toddtreece** [comment]
@ryantxu requested that this be split into two branches:
* #66218 
* #66219 

closing in favor of those PRs

**Comment by DanCech** [comment]
it may make things easier to follow if we keep this PR open and use k8s-proof-of-concept as the base for the other 2 PRs, vs having the approach-specific code mixed in with the base PoC code.  Might also help keep approach-agnostic changes separate since they can be pushed here then merged into the other 2 branches.

**Pull request closed without merging**